crawl-ref/source/dat/des/portals/bazaar.des

```diff
@@ -1548,6 +1548,65 @@ xT.xxxxxxxxxxx.Tx
 xxxxxxxxxxxxxxxxx
 ENDMAP
 
+# 3 to 4 shops, with an average of 3.5
+NAME:       bazaar_kenran_forest_clearing
+TAGS:       no_rotate no_vmirror
```

Comment:
I guess that was just my personal preference of how it looks. It's definitely fine to rotate/mirror and I'll get to that in my next commit.